features/main/headers.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Headers addition
+
+  @createSchema
+  Scenario: Test Sunset header addition
+    Given there is a DummyCar entity with related colors
```

Comment:
It is not even necessary to add this line as it would work the same for an empty collection